Modules/BenchPress.Azure/Public/Confirm-ApiManagementDiagnostic.ps1

```diff
@@ -0,0 +1,57 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-ApiManagementDiagnostic {
+  <#
+    .SYNOPSIS
+      Confirms that an API Management Diagnostic exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPApiManagementDiagnostic cmdlet gets an API Management Diagnostic using the specified API
+      Diagnostic, API, API Management Service, and Resource Group names.
+
+    .PARAMETER ResourceGroupName
+      Specifies the name of the resource group under which an API Management service is deployed.
+
+    .PARAMETER ServiceName
+      Specifies the name of the deployed API Management service.
+
+    .PARAMETER Name
+      Identifier of existing diagnostic. This will return product-scope policy. This parameters is required.
```

Comment:
```suggestion
      Identifier of existing diagnostic. This will return product-scope policy. 
```

Reply:
Note that this is direct copy/paste from the Azure PowerShell documentation for `Name` for `Get-AzApiManagementDiagnostic` (same with the other two for `New-AzApiManagementContext`). I'd rather keep it in to be consistent with the Azure PowerShell documentation, but if you feel strongly enough, I'd consider changing it.